var/spack/repos/builtin/packages/npb/package.py

```diff
@@ -189,15 +191,20 @@ def edit(self, spec, prefix):
             with open('config/suite.def', 'w') as suite_def:
                 for name in names:
                     for classname in classes:
-                        # Classes C, D and E are not available for DT
-                        if name == 'dt' and classname in ('C', 'D', 'E'):
+                        # Classes C, D, E  and F are not available for DT
+                        if name == 'dt' and classname in ('C', 'D', 'E', 'F'):
                             continue
 
-                        # Class E is not available for IS
-                        if name == 'is' and classname == 'E':
-                            continue
+                        # Class E, F is not available for IS at @3.3.1
+                        # Class F is not available for IS at @3.4.1
+                        if name == 'is':
+                            if classname == 'E':
+                                if spec.satisfies('@3.3.1'):
+                                    continue
+                            if classname == 'F':
+                                continue
```

Comment:
Is F also not available for 3.3.1? As it is I think this will skip for both.

Also, does this PR intend to add version 3.4.1? I don't see that in the changes

Reply:
Yes, F is also not available for 3.3.1.
I'm sorry for your second question.
There was my mistake while doing PR.
I added version 3.4.1 at new commit.